runtime/interface.go

```diff
@@ -81,6 +82,18 @@ type Interface interface {
 	) bool
 }
 
+type HighLevelStorage interface {
+	Interface
+
+	// HighLevelStorageEnabled should return true
+	// if the functions of HighLevelStorage should be called,
+	// e.g. SetCadenceValue
+	HighLevelStorageEnabled() bool
+
+	// SetCadenceValue sets a value for the given key in the storage, owned by the given account.
+	SetCadenceValue(owner common.Address, key string, value cadence.Value) (err error)
```

Comment:
```suggestion
	SetCadenceValue(owner Address, key string, value cadence.Value) (err error)
```

`runtime.Address` is just an alias for `common.Address` that allows the caller to avoid depending on `common`, which is kind of internal to the runtime. The other interface methods above use `runtime.Address` as well.

Reply:
Great idea! I'm so used to using `common.Address` :-) 
Changed in 15ee18fc948a26e7d38408646a5ff84727399ead